pkgs/populate/default.nix

```diff
@@ -158,6 +158,9 @@ let
     ${rsync}/bin/rsync \
         ${optionalString (config.useChecksum or false) /* sh */ "--checksum"} \
         ${optionalString target.sudo /* sh */ "--rsync-path=\"sudo rsync\""} \
+        ${concatMapStringsSep " "
+          (pattern: /* sh */ "--exclude ${quote pattern}")
+          (config.exclude or [])} \
```

Comment:
is this or required? [] is the default anyway?

Reply:
Not really. I just made it consistent with the usage `config.useChecksum` above. Should I remove both fallbacks?

Reply:
sounds like a good idea